go/plugins/ollama/ollama.go

```diff
@@ -194,16 +196,17 @@ func (g *generator) generate(ctx context.Context, input *ai.GenerateRequest, cb
 			Stream:   stream,
 		}
 	}
-	client := &http.Client{
-		Timeout: time.Second * 30,
-	}
+	client := &http.Client{Timeout: time.Second * 30}
```

Comment:
Really minor but I encourage people to write `30 * time.Second` so that it reads as the English "30 seconds".

Reply:
Done.